### ISSUE TITLE
Build files are not created for the HDZero goggles backpack

### ISSUE THREAD

**Issue by MrAlucardDante** [ExpressLRS/ExpressLRS-Configurator#556]
## Expected Behavior

After building the VRx firmware, 4 files (firmware.bin - partitions.bin - bootloader.bin - boot_app0.bin) are generated.

## Current Behavior

After building the VRx firmware, only the firmware.bin is generated.

## Steps to Reproduce

1. Launch configurator

2. Go into the backpack section

3. Choose version (tested on 1.3.0)

4. Choose Googles category

5. Choose HDZero Goggle VRX Backpack

6. Select Wifi update

7. Build the firmware

8. When the explorer opens, only the firmware.bin (as well as the renamed version) are available

## Context

Unable to update the VRX backpack in the HDZero goggles

## Your Environment

- Operating System and version: Windows 11, Configurator 1.6.0

**Comment by MrMap** [comment]
I can confirm that the Backpack file build is broken in Configurator 1.6.0

My HDZ goggles ESP32 was updated on current FW as per today. For first time I attempted to use Backpack Headtracker. No joy.
Found an howto on Discord by "Lifeless Ditto". Concluded I should already have fulfilled the prerequisites for success, except that in his example he got five backpack files from Configurator 1.5.0 while I only got two from 1.6.0.

Today I rolled back to Configurator 1.5.0 and so I too got **five files**. I made an ESP32 FW update in the HDZero goggles using an ELRS folder containing these five files - and success!  Backpack Headtracking now works as supposed to.

Conclusion: Backpack build function was broken in the Configurator 1.6.0 release, 1.5.0 works fine.


![ELRS HDZ Backpack from 1 6 0 2023-10-16 130011](https://github.com/ExpressLRS/ExpressLRS-Configurator/assets/5843103/52cc47ca-1539-471d-9839-f668d6962e21)

![ELRS HDZ backpack from 1 5 0 2023-10-16 124743](https://github.com/ExpressLRS/ExpressLRS-Configurator/assets/5843103/cf280b0d-b5f6-4e01-a440-9df3a091a5d5)
